### PR TITLE
feat: event type tabs platform wrapper

### PR DESCRIPTION
**Pull request by ibex088**
## What does this PR do?

* added platform wrappers for Limits tab
* added platform wrapper for Recurring tab

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A-I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

platform example app

### CODE REVIEW

File: packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx [hunk 1]
```diff
@@ -24,11 +24,11 @@ export default function RecurringEventController({
   const [recurringEventState, setRecurringEventState] = useState<RecurringEvent | null>(
     formMethods.getValues("recurringEvent")
   );
-  /* Just yearly-0, monthly-1 and weekly-2 */
+
   const recurringEventFreqOptions = Object.entries(Frequency)
     .filter(([key, value]) => isNaN(Number(key)) && Number(value) < 3)
     .map(([key, value]) => ({
-      label: t(`${key.toString().toLowerCase()}`, { count: recurringEventState?.interval }),
+      label: t(`${key.toString().toLowerCase()}_one`),
```
Comment (ThyMinimalDev) [inline packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx:31]: could you give some context on this change ?

Comment (ThyMinimalDev) [reply packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx:31]: is there any impact on web ?

Comment (ThyMinimalDev) [reply packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx:31]: we can use the hook useIsPlatform if needed to have a difference between the two

Comment (SomayChauhan) [reply packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx:31]: > is there any impact on web ?
tested, web works fine

File: apps/web/playwright/signup.e2e.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { expect, Page } from "@playwright/test";
```
Comment (SomayChauhan) [inline apps/web/playwright/signup.e2e.ts:1]: linting change

File: packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx (hunk 1 above)
Comment (SomayChauhan) [reply packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx]: rolled back

**Review by ThyMinimalDev — APPROVED** [review]

**Pull request merged**